client/scss/components/_body_content.scss

```diff
@@ -101,6 +102,7 @@ $center-width-xlarge: (6/6) * 100%;
     margin-right: calc((#{$narrow-width} + #{map-get($gutter-width, 'large')}) * -1);
     padding: 0 map-get($container-padding, 'large');
     padding-top: $vertical-space-unit;
+    padding-bottom: $vertical-space-unit * 2;
```

Comment:
I could put the spacing classes in here - but that would be a bit of a refactor, I'd rather just this for now.

Potentially in the future we could have something like `spacingClassesFor(el)`?